pyproject.toml

```
@@ -36,7 +36,7 @@ classifiers = [
 ]
 # add your package dependencies here
 dependencies = [
-    "gemmi",
+    "gemmi==0.7.0",
```

```suggestion
    "gemmi>=0.7.0",
```

updating here as libraries should try not to limit compatibility across the ecosystem too much, if worried about breaking changes I'd recommend pinning in your application code 🙂